### PR TITLE
feat: all non-major support

### PR DESCRIPTION
**Pull request by ahammond**
- chore: first test title, then if merged
- clarify when PR is already merged
- feat: all non-major support

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -56,17 +56,18 @@ export async function script(
     for (const pr of prs) {
       const { id, title, merged_at, html_url, draft } = pr;
 
-      // Is it already handled?
-      if (title.startsWith(expectedTitle) && merged_at) {
+      if (!title.startsWith(expectedTitle)) {
```
Comment (ahammond) [inline script.js:59]: Using startsWith to address the case when merge renames PR to add `(#123)` suffix

**Review by jeisson-clickup — APPROVED** [review]

File: script.js [hunk 2]
```diff
@@ -56,17 +56,18 @@ export async function script(
...
+        continue; // This is not hte PR we're looking for.
```
Comment (nathanhruby) [inline script.js:60]: continue or return?

**Review by nathanhruby — APPROVED** [review]
added a comment instead of review, sorry, check there

File: script.js (hunk 2 above)
Comment (ahammond) [reply script.js:63]: Continue. We still haven't found the correct PR, so try the next one.

**Pull request merged**